source/MaterialXView/Handlers/ViewHandler.cpp

```diff
@@ -0,0 +1,297 @@
+#include <MaterialXView/Handlers/ViewHandler.h>
```

Comment:
Misc view computations. Will sync with Jonathan Stone once there is some place to move some of the math here. Re-purposed some of the public ILM base code for now.